internal/dcache/rpc/client/client_pool.go

```diff
@@ -99,6 +169,13 @@ func (cp *clientPool) getNodeClientPool(nodeID string) (*nodeClientPool, error)
 		err := ncPool.createRPCClients(cp.maxPerNode)
 		if err != nil {
 			log.Err("clientPool::getNodeClientPool: createRPCClients(%s) failed: %v", nodeID, err)
+
+			//
+			// Add to negativeClients map to prevent creating new RPC clients to the node ID by other
+			// threads till the negative timeout expires.
```

Comment:
add,
Note that createRPCClients() failure indicates some transport problem or the node/blobfuse is down.